README.md

```diff
@@ -100,84 +100,7 @@ Jupyter Notebooks usage examples without any local installation are provided in
 
 ## Building CIL from source code
 
-### Getting the code
```

Comment:
Should probably link to the build instructions here

Reply:
it does later at line 103

Reply:
doh